front/public/swagger.json

```diff
@@ -15,7 +15,9 @@
       "get": {
         "summary": "Get an app run",
         "description": "Retrieve a run for an app in the workspace identified by {wId}.",
-        "tags": ["Apps"],
```

Comment:
All these changes were generated because I launched `npm run docs` locally.

Reply:
OK so basically I ran `npm run docs` which generated a lot of changes on this file, but this does not pass the linter, so I linted it, and it put it exactly back as it was before. 😄